fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/expressions/KeyExpression.java

```diff
@@ -230,6 +230,15 @@ default boolean hasLosslessNormalization() {
         return !createsDuplicates();
     }
 
+    /**
+     *
+     * @return
+     */
```

Comment:
I assume this was accidentally left blank. I'm not sure how important it is that internal APIs have Javadocs, but this should probably be removed if not filled in

Reply:
done.